aggregations/db_tables/daily_new_accounts_by_app_count.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range, time_json
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyNewAccountsByAppCount(PeriodicAggregations):
+    def dependencies(self) -> list:
+        return ['near_ecosystem_entities']
+
+    @property
+    def sql_create_table(self):
+        return '''
+            CREATE TABLE IF NOT EXISTS daily_new_accounts_by_app_count
+            (
+                collected_for_day  DATE NOT NULL,
+                app_id TEXT NOT NULL,
+                new_accounts INTEGER NOT NULL,
+                PRIMARY KEY (collected_for_day, app_id )
+            );
+            CREATE INDEX IF NOT EXISTS daily_new_accounts_by_app_count_idx
+                ON daily_new_accounts_by_app_count (collected_for_day, new_accounts DESC)
+        '''
+
+    @property
+    def sql_drop_table(self):
+        return '''
+            DROP TABLE IF EXISTS daily_new_accounts_by_app_count
+        '''
+
+    @property
+    def sql_select(self):
+        all_apps = '''
+            SELECT token, slug as app_id
+            FROM public.near_ecosystem_entities e, unnest(string_to_array(e.contract, ', ')) s(token)
+            WHERE app = TRUE AND length(contract)>0
+        '''
+
+        with self.analytics_connection.cursor() as analytics_cursor:
+            analytics_cursor.execute(all_apps)
+            app_contracts = analytics_cursor.fetchall()
+
+        string = ''
+        for index, tuple in enumerate(app_contracts):
+#            contract = ''
+            contract = str(tuple[0])
+            app = str(tuple[1])
```

Comment:
Aren't those already strings? Why do we need to `str()` those values?

Reply:
removed